.werft/build.ts

```diff
@@ -40,12 +40,6 @@ export async function build(context, version) {
      */
     werft.phase("prepare");
 
-    const werftImg = shell.exec("cat .werft/build.yaml | grep dev-environment").trim().split(": ")[1];
-    const devImg = shell.exec("yq r .gitpod.yml image").trim();
-    if (werftImg !== devImg) {
-        werft.fail('prep', `Werft job image (${werftImg}) and Gitpod dev image (${devImg}) do not match`);
-    }
-
```

Comment:
See comment above - please revert

Reply:
I don't understand the value of proactively failing at this stage.

What problem is this check trying to solve? And how bad would it be to just let this problem happen and fix it then?

Reply:
The value of pro-actively failing at this stage is to prevent configuration drift. Clearly the check was not exhaustive enough.

Reply:
done